packages/wallet/backend/migrations/20230728101910_update_transactions_columns.js

```diff
@@ -8,7 +8,7 @@ exports.up = function (knex) {
     await knex.raw(
       [
         `ALTER TABLE "transactions" DROP CONSTRAINT IF EXISTS "transactions_status_check";`,
-        `ALTER TABLE "transactions" ADD CONSTRAINT "transactions_status_check" CHECK ("status" = ANY (ARRAY['PENDING'::text, 'COMPLETED'::text, 'REJECTED'::text, 'EXPIRED'::text]))`
+        `ALTER TABLE "transactions" ADD CONSTRAINT "transactions_status_check" CHECK ("status" = ANY (ARRAY['PENDING'::text, 'COMPLETED'::text, 'REJECTED'::text, 'EXPIRED'::text, 'DELETED'::text]))`
```

Comment:
it should be a separate migration

Reply:
also a new migration should be added to mark as deleted the transactions linked to existent inactive wallet addresses

Reply:
@rico191013 the transactions linked to inactive wallet addresses should be marked as DELETED in the migration